grml-debootstrap

```diff
@@ -1427,60 +1426,60 @@ prepare_vm() {
 
   ORIG_TARGET="$TARGET" # store for later reuse
 
+  case "$ARCH" in
+    i386|amd64) true;;
+    arm64)      VMEFI=1;;
+  esac
+
+  if [ "$ARCH" = 'arm64' ]; then
+    # arm64 support largely only exists for GPT
+    VMEFI=1
+  fi
```

Comment:
lines 1429 until 1432 are doing the same as 1434 until 1437, nor?

Also, we set `VMEFI ` whenever running on arm64, feels strange?

Reply:
Indeed they are, I must have missed the lines above while getting rid of the `ARM_EFI_TARGET` variable.

Reply:
I don't believe setting `VMEFI=1` any time we're running on arm64 is wrong after all. VMEFI is set whenever running on arm64 **if** `prepare_vm` actually gets far enough to set that variable (which requires the `--vm` argument to be passed). ARM64 will always use EFI on virtual machines at the moment, and the VMEFI code path is perfectly suitable for ARM64 to use for installing its bootloader ,so it's easiest to merge the code paths. This is what lets us get rid of the problematic `ARM_EFI_TARGET` variable.

The duplicate code was definitely just wrong though, my next push will remove that.